diff --git a/node_modules/react-native-worklets/android/build.gradle b/node_modules/react-native-worklets/android/build.gradle
index e189e1c..ce19d38 100644
--- a/node_modules/react-native-worklets/android/build.gradle
+++ b/node_modules/react-native-worklets/android/build.gradle
@@ -226,7 +226,8 @@ android {
                         "-DWORKLETS_VERSION=${WORKLETS_VERSION}",
                         "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON",
                         "-DWORKLETS_FEATURE_FLAGS=${WORKLETS_FEATURE_FLAGS}",
-                        "-DHERMES_V1_ENABLED=${HERMES_V1_ENABLED}"
+                        "-DHERMES_V1_ENABLED=${HERMES_V1_ENABLED}",
+                        "-DCMAKE_SUPPRESS_REGENERATION=ON"
                 abiFilters(*reactNativeArchitectures())
                 targets("worklets")
             }
